src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -151,9 +151,8 @@ private void CallDeconstruct(BoundDeconstructionAssignmentOperator node, BoundDe
 
             var outParameters = outParametersBuilder.ToImmutableAndFree();
 
-            // invoke Deconstruct
-            var invokeDeconstruct = MakeCall(syntax, PlaceholderReplacement(deconstruction.TargetPlaceholder), deconstruction.DeconstructMemberOpt, outParameters, deconstruction.DeconstructMemberOpt.ReturnType);
-            stores.Add(invokeDeconstruct);
+            // invoke Deconstruct with placeholders replaced by locals
+            stores.Add((BoundExpression)Visit(deconstruction.DeconstructInvocationOpt));
```

Comment:
Can use `VisitExpression(deconstruction.DeconstructInvocationOpt)` to avoid needing to cast the result.